packages/application-components/src/components/public-page-layout/public-page-layout.tsx

```diff
@@ -78,16 +80,16 @@ const PublicPageLayout: FC<TProps> = (props) => {
               <div
                 css={css`
                   color: ${customProperties.colorSurface};
+                  text-align: ${themedValue('left', 'center')};
                 `}
               >
                 {props.welcomeMessage}
               </div>
             </Text.Headline>
           </ContainerColumn>
         )}
-        <Spacings.Stack scale="s">
+        <Spacings.Stack scale={themedValue('xs', 'l')}>
```

Comment:
I wanted to double check if the changed value for the default theme is also intentional.

Reply:
Nope. An 🦅 👁️ as usual. Thanks!